build.sbt

```diff
@@ -1,6 +1,7 @@
 ThisBuild / scalaVersion := "3.3.0"
 
 wartremoverWarnings ++= Warts.all
+wartremoverWarnings -= Wart.Equals
```

Comment:
I think we are old enough of an age to understand the difference between comparison by value and by reference, so I disabled the the warning on ==